Migrating to 11.4.5 result error Module not found: Error: Can't resolve 'highcharts-more' in '.../node_modules/highcharts/modules'

#### Expected behaviour
patch version should be backward compatible

#### Actual behaviour
error in ./node_modules/highcharts/modules/solid-gauge.js  
error Module not found: Error: Can't resolve 'highcharts-more' in '.../node_modules/highcharts/modules'

we did not manage to upgrade highcharts since 11.4.1. typing issues, resolution issue and more. 

#### Live demo with steps to reproduce
<!-- template: https://jsfiddle.net/highcharts/LLExL/ -->

#### Product version
<!--- Highcharts, Highstock or Highmaps plus version number -->

#### Affected browser(s)

--- a/tools/gulptasks/test-docs.js
+++ b/tools/gulptasks/test-docs.js
@@ -40,8 +40,22 @@ async function checkDocsConsistency() {
         }
 
         while ((match = requiresPattern.exec(md))) {
-            const requires = match[1]
-                .replace(/^(highcharts|stock|maps|gantt)\//u, '');
+            let requires = match[1]
+                .replace(/^(stock|maps|gantt)\//u, '');
+
+            // The @require tags in the master files are relative to the npm
+            // package root (#21470)
+            if (
+                file.startsWith('ts/masters/') &&
+                requires !== 'highcharts'
+            ) {
+                if (requires.startsWith('highcharts/')) {
+                    requires = requires.replace('highcharts/', '');
+                } else {
+                    error404s.push({ file, requires });
+                }
+            }
+
             try {
                 FS.statSync(`ts/masters/${requires}.src.ts`);
             } catch (e1) {

--- a/ts/masters/modules/cylinder.src.ts
+++ b/ts/masters/modules/cylinder.src.ts
@@ -2,7 +2,7 @@
  * @license Highcharts JS v@product.version@ (@product.date@)
  * @module highcharts/modules/cylinder
  * @requires highcharts
- * @requires highcharts-3d
+ * @requires highcharts/highcharts-3d
  *
  * Highcharts cylinder module
  *

--- a/ts/masters/modules/funnel3d.src.ts
+++ b/ts/masters/modules/funnel3d.src.ts
@@ -2,7 +2,7 @@
  * @license Highcharts JS v@product.version@ (@product.date@)
  * @module highcharts/modules/funnel3d
  * @requires highcharts
- * @requires highcharts-3d
+ * @requires highcharts/highcharts-3d
  * @requires highcharts/modules/cylinder
  *
  * Highcharts funnel module

--- a/ts/masters/modules/pyramid3d.src.ts
+++ b/ts/masters/modules/pyramid3d.src.ts
@@ -2,7 +2,7 @@
  * @license Highcharts JS v@product.version@ (@product.date@)
  * @module highcharts/modules/pyramid3d
  * @requires highcharts
- * @requires highcharts-3d
+ * @requires highcharts/highcharts-3d
  * @requires highcharts/modules/cylinder
  * @requires highcharts/modules/funnel3d
  *

--- a/ts/masters/modules/solid-gauge.src.ts
+++ b/ts/masters/modules/solid-gauge.src.ts
@@ -2,7 +2,7 @@
  * @license Highcharts JS v@product.version@ (@product.date@)
  * @module highcharts/modules/solid-gauge
  * @requires highcharts
- * @requires highcharts-more
+ * @requires highcharts/highcharts-more
  *
  * Solid angular gauge module
  *

